src/test/mir-opt/const_prop/invalid_constant.main.ConstProp.diff

```diff
@@ -24,29 +24,19 @@
   
       bb0: {
           StorageLive(_1);                 // scope 0 at $DIR/invalid_constant.rs:21:9: 21:22
--         _1 = const { InvalidChar { int: 0x110001 } }; // scope 0 at $DIR/invalid_constant.rs:21:25: 21:64
-+         _1 = const InvalidChar { int: 1114113_u32, chr: {transmute(0x00110001): char} }; // scope 0 at $DIR/invalid_constant.rs:21:25: 21:64
+          _1 = const { InvalidChar { int: 0x110001 } }; // scope 0 at $DIR/invalid_constant.rs:21:25: 21:64
                                            // mir::Constant
                                            // + span: $DIR/invalid_constant.rs:21:25: 21:64
--                                          // + literal: Const { ty: InvalidChar, val: Unevaluated(main::{constant#0}, [main::InvalidChar], None) }
-+                                          // + literal: Const { ty: InvalidChar, val: Value(Scalar(0x00110001)) }
+                                           // + literal: Const { ty: InvalidChar, val: Unevaluated(main::{constant#0}, [main::InvalidChar], None) }
           StorageLive(_2);                 // scope 1 at $DIR/invalid_constant.rs:28:9: 28:21
           StorageLive(_3);                 // scope 1 at $DIR/invalid_constant.rs:28:25: 28:46
           (_3.0: u32) = const 4_u32;       // scope 1 at $DIR/invalid_constant.rs:28:25: 28:46
--         _2 = [move _3];                  // scope 1 at $DIR/invalid_constant.rs:28:24: 28:47
-+         _2 = [const InvalidTag { int: 4_u32, e: Scalar(0x00000004): E }]; // scope 1 at $DIR/invalid_constant.rs:28:24: 28:47
-+                                          // mir::Constant
-+                                          // + span: $DIR/invalid_constant.rs:28:24: 28:47
-+                                          // + literal: Const { ty: InvalidTag, val: Value(Scalar(0x00000004)) }
+          _2 = [move _3];                  // scope 1 at $DIR/invalid_constant.rs:28:24: 28:47
```

Comment:
This was testing pretty printing of invalid constants, but it looks like those are not evaluated any more. Is this intentional?

Could you update tests if possible? If not, there is probably little reason to keep those tests at all.